src/colorizer/canvas/elements/annotations.ts

```diff
@@ -2,7 +2,7 @@ import { Vector2 } from "three";
 
 import { BaseRenderParams, EMPTY_RENDER_INFO, RenderInfo } from "../types";
 
-import { LabelData } from "../../AnnotationData";
+import { LabelData, LabelType } from "../../AnnotationData";
```

Comment:
![image](https://github.com/user-attachments/assets/79c5940f-ff2a-4c6e-84ba-ab3913de2d0a)

^ preview of what the annotation rendering looks like. Before, all annotations were rendered as the round boolean markers, but now, any annotation with a text value field (integers or custom) are rendered as rectangles with text inside.